.github/workflows/php-test.yml

```diff
@@ -46,7 +46,7 @@ jobs:
         wp: [ 'latest' ]
         include:
           - php: '7.4'
-            wp: '6.3'
+            wp: '6.4'
```

Comment:
@mukeshpanchal27 @swissspidy Strange that this is not failing in `trunk` 🤔 

But anyway, great to fix here.